plugins/modules/zos_copy.py

```diff
@@ -1454,7 +1465,7 @@ def get_file_record_length(file):
     """
     max_line_length = 0
 
-    with open(file, "r") as src_file:
+    with open(file, "r", encoding="utf-8") as src_file:
```

Comment:
I have a question, so `get_file_record_length` is used to determine record length of a file should we enter into that use case. My question is you open in utf-8 always but if the source is 1047, python would try to read its utf-8. I thought opening a file with encoding is just overriding the default behavior which detects the systems locale but the file's actual encoding has to match what you pass as encoding.

In other words, can you always use utf-8 when the source could be another encoding or am I missing something in the code.

Reply:
I might have figured it out, looks like you normalize on this block of code:
```
                    os.close(fd)
                    encode_utils.uss_convert_encoding(
                        new_src,
                        converted_src,
                        src_tag,
                        "UTF-8"
                    )
```

Reply:
Yes, before getting to `get_file_record_length` the file gets normalized.